shell/platform/darwin/ios/framework/Source/FlutterPlatformPlugin.mm

```diff
@@ -149,11 +149,21 @@ - (void)handleMethodCall:(FlutterMethodCall*)call result:(FlutterResult)result {
   } else if ([method isEqualToString:@"Share.invoke"]) {
     [self showShareViewController:args];
     result(nil);
+  } else if ([method isEqualToString:@"ContextMenu.showSystemContextMenu"]) {
+    [self showSystemContextMenu:args];
+    result(nil);
   } else {
     result(FlutterMethodNotImplemented);
   }
 }
 
+- (void)showSystemContextMenu:(NSDictionary*)args {
```

Comment:
It should be easy to add. Let me try!

Reply:
Just added `ContextMenu.hideSystemContextMenu` (no params). Feel free to play around. 

> Or maybe it doesn't matter since it's only related to development and not a release build.

If it's easy for you, I'm open to just fit it here. But if it's hard, we can also leave it, and I can create an issue to track that (no strong opinion)

Reply:
Ok I'll play around with the hot restart thing and see if I can fix it, otherwise we'll open an issue.

Reply:
Thanks for the quick turnaround on hideSystemContextMenu. I'll try it out and post back.